build/build.sh

```diff
@@ -42,31 +42,32 @@ pandoc --verbose \
   --output=output/manuscript.html \
   $INPUT_PATH
 
-# Create PDF output
-echo "Exporting PDF manuscript"
-ln -s content/images images
-pandoc \
-  --from=markdown \
-  --to=html5 \
-  --pdf-engine=weasyprint \
-  --pdf-engine-opt=--presentational-hints \
-  --filter=pandoc-fignos \
-  --filter=pandoc-eqnos \
-  --filter=pandoc-tablenos \
-  --bibliography=$BIBLIOGRAPHY_PATH \
-  --csl=$CSL_PATH \
-  --metadata link-citations=true \
-  --webtex=https://latex.codecogs.com/svg.latex? \
-  --css=webpage/github-pandoc.css \
-  --output=output/manuscript.pdf \
-  $INPUT_PATH
-rm -r images
+# Create PDF output (unless BUILD_PDF environment variable equals "false")
+if [ "$BUILD_PDF" != "false" ]; then
+  echo "Exporting PDF manuscript"
+  ln -s content/images images
```

Comment:
@agitter this could also be a good PR to fix the image symlink issues.

So I think the main issue we've had is when the pandoc command fails, so `rm -r images` never occurs. This can lead to a leftover images symlink that prevents causes subsequent `ln -s content/images images` commands to fail.

One option would be to add the `-f` option to `ln`, causing it to overwrite an existing `images` symlink. However, there may be a risk if users somehow try running these commands in a different directory, this could delete their `content/images` directory and contents perhaps.

Reply:
That option could work.  Many users may already have the `content/images` in version control so accidental deletion wouldn't be catastrophic.  But for anyone who didn't have them versioned and accidentally removed content with the script it would be a major problem.

I haven't come up with a better option though.  Perhaps the script could test whether `images` already exists and if it does, test whether it is a symbolic link.  There should be minimal harm in overwriting or removing a symbolic link.